src/test/groovy/com/intuit/graphql/orchestrator/integration/federation/EntityTypeSpec.groovy

```diff
@@ -322,4 +325,149 @@ class EntityTypeSpec extends BaseIntegrationTestSpecification {
         reviewResolver.get("rating") == 5
         reviewResolver.get("comment") == "test review"
     }
+
+    def "Entity with multiple keys is stitched"() {
```

Comment:
NIT: I'm thinking of change the title/function name to "Entity with multiple key directives is stitched".

Reply:
sounds good